tracing/setup.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package tracing
+
+import (
+	"fmt"
+
+	"go.uber.org/zap"
+
+	"knative.dev/pkg/configmap"
+	"knative.dev/pkg/tracing/config"
+)
+
+// setupPublishing sets up trace publishing for the process. Note that other pieces
+// still need to generate the traces, this just ensures that if generated, they are collected
+// appropriately. This is normally done by using tracing.HTTPSpanMiddleware as a middleware HTTP
+// handler.
+func setupPublishing(serviceName string, logger *zap.SugaredLogger) *OpenCensusTracer {
+	return NewOpenCensusTracer(WithExporter(serviceName, logger))
+}
```

Comment:
Given this is an internal method consisting of a single line — why do we need it?